include/dos_system.h

```diff
@@ -190,38 +190,6 @@ class DOS_Device : public DOS_File {
 	Bitu devnum;
 };
 
-class localFile : public DOS_File {
-public:
-	localFile(const char* name, const std_fs::path& path,
-	          const NativeFileHandle handle, const char* basedir,
-	          bool _read_only_medium);
-	localFile(const localFile&)            = delete; // prevent copying
-	localFile& operator=(const localFile&) = delete; // prevent assignment
-	bool Read(uint8_t* data, uint16_t* size) override;
-	bool Write(uint8_t* data, uint16_t* size) override;
-	bool Seek(uint32_t* pos, uint32_t type) override;
-	void Close() override;
-	uint16_t GetInformation() override;
-	bool IsOnReadOnlyMedium() const override { return read_only_medium; }
-	const char* GetBaseDir() const
-	{
-		return basedir;
-	}
-	std_fs::path GetPath() const
-	{
-		return path;
-	}
-	NativeFileHandle file_handle = InvalidNativeFileHandle;
-
-private:
-	void MaybeFlushTime();
-	const std_fs::path path = {};
-	const char* basedir     = nullptr;
-
-	const bool read_only_medium = false;
-	bool set_archive_on_close   = false;
-};
-
```

Comment:
> I was running into circular header dependencies for what I wanted to do next.

Oh yeah, header files suck hard in C/C++. And the inability of two files to export/import symbols between themselves without a third intermediary.